tests/test_run_simulation.py

```diff
@@ -327,28 +329,28 @@ def test_discretize_gamma(self):
         alpha = 1.25
         ncat = 4
         expected = np.array([0.09199853806558903, 0.27043066909631136, 0.5158061369385518, 1.1217646558655263])
-        result = discretize(alpha, ncat, dist=ss.gamma)
+        result = discretize(alpha, ncat, dist=ss.gamma, scale=0)
```

Comment:
Setting `scale` to zero does not make sense - this  should not produce reasonable outputs!